### PR TITLE
Autoalign

### PR DESCRIPTION
**Pull request by Noah9847**
auto align functionality and some other small limelight updates

### CODE REVIEW

File: 2024 Code/src/main/java/frc/robot/subsystems/LimelightSubsystem.java [hunk 1]
```diff
@@ -8,6 +9,9 @@
 public class LimelightSubsystem extends SubsystemBase {
   private NetworkTable limelight;
   private double[] pose = new double[6];
+  private double[] targetPose = new double[6];
+  private double[] ppose = new double[6];
```
Comment (Noah9847) [inline 2024 Code/src/main/java/frc/robot/subsystems/LimelightSubsystem.java:13]: better name

File: 2024 Code/src/main/java/frc/robot/commands/LimelightDefaultCommand.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    public void initialize() {
+    }
+
+    @Override
+    public void execute() {
+        if (m_limelight.getTv()) {
+            botPose = m_limelight.getBotPose();
+            System.out.println(Arrays.toString(botPose));
+            pose = new Pose2d(botPose[0], botPose[1],
+                    Rotation2d.fromDegrees(SubsystemContainer.swerveSubsystem.getYaw()));// new Rotation2d(botPose[5]));
```
Comment (Noah9847) [inline 2024 Code/src/main/java/frc/robot/commands/LimelightDefaultCommand.java:36]: remove comment

File: 2024 Code/src/main/java/frc/robot/commands/LimelightDefaultCommand.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+    @Override
+    public void initialize() {
+    }
+
+    @Override
+    public void execute() {
+        if (m_limelight.getTv()) {
+            botPose = m_limelight.getBotPose();
+            System.out.println(Arrays.toString(botPose));
```
Comment (Noah9847) [inline 2024 Code/src/main/java/frc/robot/commands/LimelightDefaultCommand.java:34]: remove print

File: 2024 Code/src/main/java/frc/robot/commands/AutoAlignSequentialCommand.java [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+import frc.robot.FieldConstants;
+import frc.robot.subsystems.LimelightSubsystem;
+import frc.robot.subsystems.SwerveSubsystem;
+import frc.robot.util.SubsystemContainer;
+import edu.wpi.first.hal.AllianceStationID;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj2.command.Command;
+
+/** An example command that uses an example subsystem. */
+public class AutoAlignSequentialCommand extends Command {
```
Comment (Noah9847) [inline 2024 Code/src/main/java/frc/robot/commands/AutoAlignSequentialCommand.java:16]: this can inherit from AutoAlignCommand

**Review by gurellia53 — APPROVED** [review]

**Pull request merged**